File: app/src/main/java/com/xlzhen/sharefileserver/service/BaseService.java
package com.xlzhen.sharefileserver.service;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.util.Log;

/* loaded from: classes8.dex */
public abstract class BaseService extends Service {
    @Override // android.app.Service
    public IBinder onBind(Intent intent) {
        Log.i(getClass().getName(), "..........onBind..........");
        throw new UnsupportedOperationException("Not yet implemented");
    }

    @Override // android.app.Service
    public void onCreate() {
        super.onCreate();
        Log.i(getClass().getName(), "..........onCreate..........");
    }

    @Override // android.app.Service
    public void onDestroy() {
        super.onDestroy();
        Log.i(getClass().getName(), "..........onDestroy..........");
    }

    @Override // android.app.Service, android.content.ComponentCallbacks
    public void onLowMemory() {
        super.onLowMemory();
        Log.i(getClass().getName(), "..........onLowMemory..........");
    }

    @Override // android.app.Service
    public void onRebind(Intent intent) {
        super.onRebind(intent);
        Log.i(getClass().getName(), "..........onRebind..........");
    }

    @Override // android.app.Service
    public int onStartCommand(Intent intent, int i, int i2) {
        Log.i(getClass().getName(), "..........onStartCommand..........");
        return super.onStartCommand(intent, i, i2);
    }

    @Override // android.app.Service
    public void onTaskRemoved(Intent intent) {
        super.onTaskRemoved(intent);
        Log.i(getClass().getName(), "..........onTaskRemoved..........");
    }

    @Override // android.app.Service, android.content.ComponentCallbacks2
    public void onTrimMemory(int i) {
        super.onTrimMemory(i);
        String name = getClass().getName();
        Log.i(name, "..........onTrimMemory..........>" + i);
    }

    @Override // android.app.Service
    public boolean onUnbind(Intent intent) {
        Log.i(getClass().getName(), "..........onUnbind..........");
        return super.onUnbind(intent);
    }
}

File: app/src/main/java/com/xlzhen/sharefileserver/service/BaseService.kt
package com.xlzhen.sharefileserver.service

import android.app.Service
import android.content.Intent
import android.os.IBinder
import android.util.Log

abstract class BaseService : Service() {
    override fun onBind(intent: Intent?): IBinder? {
        Log.i(javaClass.name, "..........onBind..........")
        throw UnsupportedOperationException("Not yet implemented")
    }

    override fun onCreate() {
        super.onCreate()
        Log.i(javaClass.name, "..........onCreate..........")
    }

    override fun onDestroy() {
        super.onDestroy()
        Log.i(javaClass.name, "..........onDestroy..........")
    }

    override fun onLowMemory() {
        super.onLowMemory()
        Log.i(javaClass.name, "..........onLowMemory..........")
    }

    override fun onRebind(intent: Intent?) {
        super.onRebind(intent)
        Log.i(javaClass.name, "..........onRebind..........")
    }

    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
        Log.i(javaClass.name, "..........onStartCommand..........")
        return super.onStartCommand(intent, flags, startId)
    }

    override fun onTaskRemoved(intent: Intent?) {
        super.onTaskRemoved(intent)
        Log.i(javaClass.name, "..........onTaskRemoved..........")
    }

    override fun onTrimMemory(level: Int) {
        super.onTrimMemory(level)
        Log.i(javaClass.name, "..........onTrimMemory..........>$level")
    }

    override fun onUnbind(intent: Intent?): Boolean {
        Log.i(javaClass.name, "..........onUnbind..........")
        return super.onUnbind(intent)
    }
}
